Src/Support/Google.Apis.Auth/OAuth2/ServiceAccountCredential.cs

```diff
@@ -133,13 +139,8 @@ public Initializer FromCertificate(X509Certificate2 certificate)
         /// <summary>Unix epoch as a <c>DateTime</c></summary>
         protected static readonly DateTime UnixEpoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
 
-        private readonly string id;
-        private readonly string user;
-        private readonly IEnumerable<string> scopes;
-        private readonly RsaKey key;
-
         /// <summary>Gets the service account ID (typically an e-mail address).</summary>
-        public string Id { get { return id; } }
+        public string Id { get; private set; }
```

Comment:
Do you have need the private set? If these were previously read-only fields, they can just be read-only properties, as presumably they're only set in the constructor.